backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCHParquetSuite.scala

```diff
@@ -48,6 +48,8 @@ class GlutenClickHouseTPCHParquetSuite extends GlutenClickHouseTPCHAbstractSuite
       .set("spark.sql.autoBroadcastJoinThreshold", "10MB")
       .set("spark.gluten.sql.columnar.backend.ch.use.v2", "false")
       .set("spark.gluten.supported.scala.udfs", "my_add")
+    // .set("spark.sql.planChangeLog.level", "error")
+    // .set("spark.gluten.sql.commonSubexpressionEliminate", "true")
```

Comment:
remove these ?

Reply:
I'd like keep it for possible debugging in the future

Reply:
`spark.gluten.sql.commonSubexpressionEliminate` is already used with `withSQLConf`, can remove them